ts/IterableInboxClickableRow.tsx

```diff
@@ -0,0 +1,101 @@
+"use strict"
+import React, { Component, useState } from "react"
+import IterableInboxMessageListItem from "./IterableInboxMessageListItem"
+import { 
+   View, 
+   Text, 
+   StyleSheet, 
+   TouchableOpacity 
+} from "react-native"
+
+type MessageClickableRowProps = {
+   index: number,
+   message: {[key: string]: any},
+   handleMessageSelect: Function,
+   last: boolean 
+}
+
+const IterableInboxClickableRow = ({ index, message, handleMessageSelect, last }: MessageClickableRowProps) => {
+   const [active, setActive] = useState(false)
+
+   return(
+      <TouchableOpacity
+         style={
+            last
+               ? ((active) ? styles.pressedLastMessageCell : styles.lastMessageCell)
+               : ((active) ? styles.pressedMessageCell : styles.messageCell)
```

Comment:
this is a personal nit, but I'd argue that a ternary inside of a ternary is hard to follow at first glance: 

how do you feel about creating a couple of variables  on line 20 like... 
```
const lastStyle = active ? styles.pressedLastMessageCell : styles.lastMessageCell;
const style = active ? styles.pressedMessageCell : styles.messageCell;
``` 

and then you can turn this into `style={ last ? lastStyle : style}`

Reply:
I agree. The nested ternary did not look very good to me.